### PR TITLE
Ruby webhooks

### PR DESCRIPTION
**Pull request by plaurynovich-hubspot**
Local gem usage was removed.
Docker file was fixed.
Events list was simplified (we just show a list of events for now).

### CODE REVIEW

**Comment by atanasiuk-hubspot** [comment]
Is this a full copy of existing sample app? Or please describe what changes have been made. Thanks

File: ruby/db/mysql/ib_buffer_pool [hunk 1]
```diff
@@ -0,0 +1,253 @@
+4294967278,284
```
Comment (atanasiuk-hubspot) [inline ruby/db/mysql/ib_buffer_pool]: Do we need all this files to be committed?

**Comment by plaurynovich-hubspot** [comment]
> Is this a full copy of existing sample app? Or please describe what changes have been made. Thanks

Added some discription.

File: ruby/db/mysql/ib_buffer_pool (hunk 1 above)
Comment (plaurynovich-hubspot) [reply ruby/db/mysql/ib_buffer_pool]: Thx, looks like we should add them to the gitignore.

**Review by ksvirkou-hubspot — CHANGES_REQUESTED** [review]
Could you rm mysql files?

File: ruby/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    property_name   varchar(255)    default null,
+    property_value  varchar(255)    default null,
+    created_at      datetime        default CURRENT_TIMESTAMP
+);
+```
+Please note that event_id sent by HubSpot needs to be stored as int
+
+### Setup App
+
+Make sure you have [Docker Compose](https://docs.docker.com/compose/) and [Ngrok](https://ngrok.com/) installed.
```
Comment (atanasiuk-hubspot) [inline ruby/README.md]: Do we need Ngrok to be installed locally? Looks like it's included into `docker-compose.yml`

**Review by atanasiuk-hubspot — APPROVED** [review]
LGTM

**Pull request merged**